main.py

```diff
@@ -3,32 +3,42 @@
 from flask import Flask, request, render_template
 from src.DataFetcher import fetchDataForTickerSymbol
 
+logger = logging.getLogger("IsThisStockGood")
+
+handler = logging.StreamHandler()
+handler.setLevel(logging.WARNING)
+
+h_format = logging.Formatter('%(name)s - %(levelname)s : %(message)s')
+handler.setFormatter(h_format)
+
+logger.addHandler(handler)
+logger.level
 
 app = Flask(__name__)
 
 
 @app.route('/')
 def homepage():
-  if request.environ['HTTP_HOST'].endswith('.appspot.com'):  #Redirect the appspot url to the custom url
-    return '<meta http-equiv="refresh" content="0; url=https://isthisstockgood.com" />'
+    if request.environ['HTTP_HOST'].endswith('.appspot.com'):  #Redirect the appspot url to the custom url
```

Comment:
Can we revert the whitespace changes from lines 22 onward? (Seems like the only real changes to this file are the logger lines `6-15`.

Reply:
I see no reason to keep them if there was no real changes, so yes